### ISSUE TITLE
Api documentation additions

### ISSUE THREAD

**Issue by miami78** [DevCEldoret/devceldoret-backend#5]
* add apiary link in the readme as well as the VSCode extension.
* explain how people can make modifications to the `.apib `file

**Comment by Kenchir** [comment]
Hi, I am not getting what is supposed to be done at backend. Maybe someone should note down the deliverables.

**Comment by miami78** [comment]
@Kenchir we are working on the readme and contributions.md  so as to make it easier for people to contribute, the project scope hasn't been discussed yet that's why we don't have any APIs up or features being worked on right now. We will have a call to verify all that and I will open a couple of issues where all can contribute

**Comment by VerumEmanio** [comment]
@miami78 Is this API documentation a good first project for a beginner? I'm a technical writer who knows some JavaScript, and I want to get some hands-on experience writing REST API documentation.

**Comment by MarvinKweyu** [comment]
@VerumEmanio , the documentation has been assigned already, unfortunately.

**Comment by miami78** [comment]
@VerumEmanio yes it is but unfortunately this issue was already assigned to someone else,keep checking the issues tab we will be updating new issues soon.....
